exonum/src/encoding/fields.rs

```diff
@@ -68,10 +68,15 @@ pub trait Field<'a> {
     }
 }
 
-/// implement field for all types that has writer and reader functions
+/// Implements the [`Field`] trait for all types that have writer and reader functions.
```

Comment:
I find "all" confusing here; the macro implements the `Field` trait for a single specified type. Suggest to reword to

> Implements the `Field` trait for a type that has writer and reader functions.

Similar concerns for other macro descriptions as well.